File: examples/solver.js
const axios = require('axios');

const apiUrl = 'http://45.140.188.54:6200/solve';
const apiKey = 'your api key';
const uid = '123456789';
const proxy = 'can be left empty';

const payload = {
  apiKey: apiKey,
  uid: uid
};

axios.post(apiUrl, payload)
  .then(response => {
    console.log('Response:', response.data);
  })
  .catch(error => {
    console.error('Error:', error.message);
  });

File: examples/solver.py
import requests

api_url = 'http://45.140.188.54:6200/solve'
api_key = 'your api key'
uid = '123456789'

payload = {
    'apiKey': api_key,
    'uid': uid
}

try:
    response = requests.post(api_url, json=payload)
    print('Response:', response.json())
except requests.exceptions.RequestException as error:
    print('Error:', error)
